test/lib/smart-survey-api.test.js

```diff
@@ -2,106 +2,60 @@
 
 const sinon = require('sinon');
 const chai = require('chai').use(require('sinon-chai'));
-chai.should();
 const proxyquire = require('proxyquire');
+chai.should();
+chai.use(require('chai-as-promised'));
```

Comment:
Nice one, I've now done that.  I used lev-web as the example, thanks @daniel-ac-martin